plugins/plugin-chart-echarts/src/Gauge/EchartsGauge.tsx

```diff
@@ -16,10 +16,76 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-import React from 'react';
-import { EchartsProps } from '../types';
+import React, { useCallback } from 'react';
+import { GaugeChartTransformedProps } from './types';
 import Echart from '../components/Echart';
+import { EventHandlers } from '../types';
 
-export default function EchartsGauge({ height, width, echartOptions }: EchartsProps) {
-  return <Echart height={height} width={width} echartOptions={echartOptions} />;
+export default function EchartsGauge({
+  height,
+  width,
+  echartOptions,
+  setDataMask,
+  labelMap,
+  groupby,
+  selectedValues,
+  formData,
```

Comment:
```suggestion
  formData: { emitFilter }
```

Can probably expand the input args directly if `formData` is not going to be used elsewhere.

Reply:
> LGTM. Although I can't imagine why would someone place multiple ticks on a gauge.... 😱

I do think this kind of case is relatively rare, but it can provide this ability to users.😂
Also, we plan to add x-filter functionality to all Echarts.